src/Throttlers/CacheThrottler.php

```diff
@@ -147,6 +147,20 @@ public function count()
         return $this->number;
     }
 
+    /**
+     * Get the time until the end of current duration.
+     *
+     * @return int
+     */
+    public function ttl()
+    {
+        if (!$this->store instanceof RedisStore) {
+            throw new \LogicException('The ttl() method can only be called for Redis backed throttlers');
```

Comment:
I don't think we should throw an exception here, since it means the code isn't driver agnostic anymore.

Reply:
So what would be better than to indicate the ttl() method is not supported? Maybe return a negative number?